dotcom-rendering/src/web/lib/getZIndex.test.tsx

```diff
@@ -1,27 +1,30 @@
-import { getZIndex } from './getZIndex';
+import { getZIndex, ZIndex } from './getZIndex';
 
 describe('getZIndex', () => {
-	it('gets the correct zindex for group and sibling', () => {
-		expect(getZIndex('sticky-video-button')).toBe('z-index: 21;');
-		expect(getZIndex('sticky-video')).toBe('z-index: 20;');
-		expect(getZIndex('banner')).toBe('z-index: 19;');
-		expect(getZIndex('dropdown')).toBe('z-index: 18;');
-		expect(getZIndex('burger')).toBe('z-index: 17;');
-		expect(getZIndex('expanded-veggie-menu-wrapper')).toBe('z-index: 16;');
-		expect(getZIndex('expanded-veggie-menu')).toBe('z-index: 15;');
-		expect(getZIndex('stickyAdWrapperLabsHeader')).toBe('z-index: 14;');
-		expect(getZIndex('stickyAdWrapper')).toBe('z-index: 13;');
-		expect(getZIndex('stickyAdWrapperNav')).toBe('z-index: 12;');
-		expect(getZIndex('editionDropdown')).toBe('z-index: 11;');
-		expect(getZIndex('toast')).toBe('z-index: 10;');
-		expect(getZIndex('onwardsCarousel')).toBe('z-index: 9;');
-		expect(getZIndex('searchHeaderLink')).toBe('z-index: 8;');
-		expect(getZIndex('TheGuardian')).toBe('z-index: 7;');
-		expect(getZIndex('headerWrapper')).toBe('z-index: 6;');
-		expect(getZIndex('articleHeadline')).toBe('z-index: 5;');
-		expect(getZIndex('immersiveBlackBox')).toBe('z-index: 4;');
-		expect(getZIndex('bodyArea')).toBe('z-index: 3;');
-		expect(getZIndex('rightColumnArea')).toBe('z-index: 2;');
-		expect(getZIndex('mainMedia')).toBe('z-index: 1;');
+	const tests: Array<[ZIndex, number]> = [
+		['sticky-video-button', 21],
+		['sticky-video', 20],
+		['banner', 19],
+		['dropdown', 18],
+		['burger', 17],
+		['expanded-veggie-menu-wrapper', 16],
+		['expanded-veggie-menu', 15],
+		['stickyAdWrapperLabsHeader', 14],
+		['stickyAdWrapper', 13],
+		['stickyAdWrapperNav', 12],
+		['editionDropdown', 11],
+		['toast', 10],
+		['onwardsCarousel', 9],
+		['searchHeaderLink', 8],
+		['TheGuardian', 7],
+		['headerWrapper', 6],
+		['articleHeadline', 5],
+		['immersiveBlackBox', 4],
+		['bodyArea', 3],
+		['rightColumnArea', 2],
+		['mainMedia', 1],
+	];
+	it.each(tests)('Get correct index for %s : %s', (name, index) => {
+		expect(getZIndex(name)).toBe(`z-index: ${index};`);
```

Comment:
[test.each](https://jestjs.io/docs/api#testeachtablename-fn-timeout) fails and succeeds in a more transparent way.